examples/complete/versions.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  required_version = "~> 1.7"
```

Comment:
Why 1.7? Do we want to pin this to the version in aqua?

Reply:
No specific requirements. Consolidated.